test/integration/datadog_consul/serverspec/consul_spec.rb

```diff
@@ -7,7 +7,7 @@
 set :backend, :exec
 set :path, '/sbin:/usr/local/sbin:$PATH'
 
-AGENT_CONFIG = '/etc/dd-agent/conf.d/consul.yaml'.freeze
+AGENT_CONFIG = '/etc/datadog-agent/conf.d/consul.yaml'.freeze
```

Comment:
I know this location is supported, but wouldn't it maybe be better to use the default expected location: `/etc/datadog-agent/conf.d/consul.d/conf.yaml`?

Also, have we entirely dropped A5 support? Would we want to keep testing for backward compatibility?

Reply:
Those are old kitchen tests manually started while working on the cookbook.
It looks like they're not really up-to-date (e.g. logs value), I don't think we mind preserving backward compatibility here.